docs/source/using/profiles.adoc

```diff
@@ -39,7 +39,7 @@ There are two ways to create a new profile.
 [NOTE]
 ====
 Profile name can only consist of alphanumeric characters.
-The use of underscore ( *_* ) and hyphen ( *-* ) are allowed as separator.
+The use of hyphen ( *-* ) is allowed as separator. The regex `^[a-zA-Z0-9]+[a-zA-Z0-9-]*$` is used to validate profile names.
```

Comment:
I'm not sure that we need to include the regular expression used within the documentation. We're already describing which characters are valid for profile names, and I think that's ideal. :)

Basically, I would remove the added sentence -- I'm not convinced that it really contributes anything to the documentation.

Reply:
> Basically, I would remove the added sentence -- I'm not convinced that it really contributes anything to the documentation.

+1. Its really not good to add the code behind in doc. This sentence ```The regex `^[a-zA-Z0-9]+[a-zA-Z0-9-]*$` is used to validate profile names.``` should be completely removed.

Reply:
-1 for ` The regex `^[a-zA-Z0-9]+[a-zA-Z0-9-]*$` is used to validate profile names.` in the doc section. We already say `Profile name can only consist of alphanumeric characters.`

Reply:
Alright.. removing it!